test/dkanextension/src/Drupal/DKANExtension/Context/DatasetContext.php

```diff
@@ -129,6 +129,10 @@ public function iShouldSeeTheFirstDatasetListInOrder($number, $orderby, $sortdir
     }
 
     switch ($orderby) {
+      case 'Date created':
```

Comment:
@janette looking at this test code, I'm pretty sure that it's going to give a false positive. Where is it actually checking the contents of the first four datasets against expected values? All the assertion at the end of the test seems to check is that at least four datasets exist.

Reply:
@dafeder do you think this check for the order of result titles does not work? https://github.com/GetDKAN/dkan/blob/a977f6b54f3a8534df2b8643faa03a199b05f392/test/dkanextension/src/Drupal/DKANExtension/Context/DatasetContext.php#L161

Reply:
LOL you're right. I misunderstood the code 🤦.